Add the `Skinning Contest: Unchained - Results` news post & amend wiki page (#11750)
ppy/osu-wiki: news/2024/2024-07-17-skinning-contest-unchained-results.md

---
layout: post
title: "Skinning Contest: Unchained - Results"
date: 2024-07-17 12:00:00 +0000
---

The voting has concluded, and it's time to put the creativity genie back in the box and check out the voting results.

![](/wiki/shared/news/2024-07-17-skinning-contest-unchained-results/Results_Out-Newspost.jpg)

> The ballot box has been chained up, and the results have been tallied up. This edition of the skinning contest has seen **1,839** community members cast a total of **4,637** votes across **28** different entries. It's time to dive in head first, see which entries topped the charts, and most importantly which one secured themselves the #1 spot.

Reviewer: ```suggestion
The ballot box has been chained up, and the results have been tallied up. This edition of the skinning contest has seen **1,839** community members cast a total of **4,637** votes across **28** different entries. It's time to dive in head first, see which entries topped the charts, and most importantly which one secured themselves the coveted #1 spot.
```

### 3rd Place - ROSULA

> Inspired by the design of [Navia](https://genshin-impact.fandom.com/wiki/Navia) from *Genshin Impact*, [Chromasia](https://osu.ppy.sh/users/7306251) has once again put themselves near the top of the list. Even though this is their 2nd time entering a skinning contest, they have quickly solidified themselves in the upper echelon of the skinning community. Their newest entry does not miss the mark either, basking the user with tastefully placed golden yellow elements across the entire skin, along with Navia as the centre piece on the song selection screen.

Reviewer: ```suggestion
Inspired by the design of [Navia](https://genshin-impact.fandom.com/wiki/Navia) from *Genshin Impact*, [Chromasia](https://osu.ppy.sh/users/7306251) has once again put themselves near the top of the list. Even though this is their 2nd time entering a skinning contest, they have quickly solidified themselves in the upper echelon of the skinning community. Their newest entry does not miss the mark either, basking users with some tastefully-placed golden yellow accents across the entire skin, along with Navia as the centre piece on the song selection screen.
```

> <div align="center">

Reviewer: ```suggestion
<div align="center" class="osu-md__paragraph">
```

    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/HpL7rWyVjP4" frameborder="0" allowfullscreen></iframe>
</div>

This skin ticks all the aesthetically pleasing boxes, but is also a very functional play skin, which is why Chromasia were able to snag themselves 3rd place with **401** votes.

![](/wiki/shared/news/2024-07-17-skinning-contest-unchained-results/3rd_badge@2x.png)

> ### 2nd Place - echoing inside

Reviewer: ```suggestion
### 2nd Place - echoing inside.
```

It's hard to talk about modern skinning without mentioning [thetasigma](https://osu.ppy.sh/users/6234482) at least once. Consistently ranking in the top 3 of all recent skinning competitions, they have proven themselves as a top tier skinner. Their newest entry, [echoing inside.](https://osu.ppy.sh/community/forums/topics/1925867), is themed around [Robin](https://honkai-star-rail.fandom.com/wiki/Robin), a character from *Honkai: Star Rail*, and thetasigma has used the character's colour palette extensively throughout the skin, with a focus on simplistic gameplay.

> <div align="center">

Reviewer: ```suggestion
<div align="center" class="osu-md__paragraph">
```

    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/9PnF9wPOsJo" frameborder="0" allowfullscreen></iframe>
</div>

Gacha gamers recognized thetasigma's efforts and gave them good RNG, which they used to pull **531** votes, no pity mechanics required.

![](/wiki/shared/news/2024-07-17-skinning-contest-unchained-results/2nd_badge@2x.png)

### 1st Place - Phosphor

[Miorine](https://osu.ppy.sh/users/12630408) has strayed away from the usual anime girl themes, and focused instead on the topic of nostalgia for their contest entry. After barely losing out on 1st place during last year's [Skin of the Year](https://osu.ppy.sh/home/news/2024-02-05-skin-of-the-year-2023-results) contest, they have once again returned with more of an 80s style skin, throwing all of us back to the times of CRT's, floppy disks and beige-painted plastics wherever you look.

> Players might even find themselves more immersed into the theme in gameplay rather than the song selection screen, with a unique take on the playable border that's completely skinned out to look like you are actually playing on a CRT. We'd also like to shout out the unique play area of mania as well, as it's been completely reskinned to look like a never-ending punch card.

Reviewer: ```suggestion
Players might even find themselves more immersed into the theme in gameplay rather than the song selection screen, with a unique take on the playable border that's completely skinned out to look like you are actually playing on a CRT. We'd also like to shout out osu!mania's unique playfield as well, as it's been completely reskinned to look like a never-ending punch card.
```

> <div align="center">

Reviewer: ```suggestion
<div align="center" class="osu-md__paragraph">
```

    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/sw1nhClFPLg" frameborder="0" allowfullscreen></iframe>
</div>

> These are only some of the reasons that the community liked this entry so much that they voted **806** times for it, which put it a whopping 275 votes above second place!

Reviewer: ```suggestion
These are only some of the reasons that the community liked this entry so much that they voted **806** times for it, which put it a whopping **275** votes above 2nd place!
```

![](/wiki/shared/news/2024-07-17-skinning-contest-unchained-results/1st_badge@2x.png)

---

> Congratulations to all the winners, and thank you to everyone who participated and voted! You can view the full results on the [contest page](https://osu.ppy.sh/community/contests/208), and discuss the results on the [discussion page](https://osu.ppy.sh/community/forums/topics/1906590?n=1). If you are interested in skinning or just the community surrounding it, feel free to check out the [skinning tutorial](https://skinship.xyz/tutorial/introduction) and join the [skinship Discord server](https://discord.skinship.xyz/)!

Reviewer: ```suggestion
Congratulations to all the winners, and thank you to everyone who participated and voted! You can view the full results on the [contest page](https://osu.ppy.sh/community/contests/208), and discuss the results on the [discussion page](https://osu.ppy.sh/community/forums/topics/1906590?n=1).

If you are interested in skinning or just the community surrounding it, feel free to check out the [skinning tutorial](https://skinship.xyz/tutorial/introduction) and join the [skinship Discord server](https://discord.skinship.xyz/)!
```
